app/admin/allocation_chart.rb

```diff
@@ -22,6 +22,9 @@
         column(I18n.t('skills'), class: 'allocated-column__last') do |allocation|
           decorated_user(allocation).skills
         end
+        column(:allocated_until_data) do |allocation|
+          allocation.end_at ? allocation.end_at&.to_time.to_i : 'Not allocated'
```

Comment:
do we need the safe operator if we're checking if it exists first?

Reply:
Actually no. I'll remove right now.